### ISSUE TITLE
Abstract nodes 

### ISSUE THREAD

**Issue by AndreOliveiraRepos** [neos/neos-ui#2338]
### Description

In the backend we have a Nodetype as Abstract then it is extended based on article types. Upon selection of the node, the ui completely shutdown to a blank page.
The extended nodetype serves so that the it can't be initialised on itself without extending based on article type.

### Steps to Reproduce

1. Add a Nodetype that is extended by an abstract nodetype.
2. On the backend, upon selection the UI shuts down.

#### Expected behavior

It should initialise an extended nodetype with a set property. The behavior without the abstract property is the following:

![chrome-capture](https://user-images.githubusercontent.com/8405325/52401189-613db700-2ac1-11e9-9d82-f01ad090bd3f.gif)

That is the correct behavior since it doesn't completely breaks functionality. 

#### Actual behavior

Adding the property abstract to the superNodeType and set at true. Upon selecting the extended Nodetype the behavior is as follow:

![break](https://user-images.githubusercontent.com/8405325/52401384-d0b3a680-2ac1-11e9-85ad-8580fa975018.gif)

Thrown console error:

![screenshot 2019-02-07 at 10 23 06](https://user-images.githubusercontent.com/8405325/52401630-6d764400-2ac2-11e9-8121-771edb53596a.png)

FilteredList is the Abstract Node.

### Affected Versions

Neos: 4.2.8

UI: 3.0.2

**Comment by dimaip** [comment]
Thanks for reporting!
Wanna give a shot at fixing this ;)

**Comment by danielkestler** [comment]
By „marking as abstract“ you mean changing the NodeType configuration yaml? This way, you would have to make sure to migrate your existing nodes to your preferred type. Not sure if this really is a UI bug ... (but of course, these white windows should not occur)

Or maybe I am not getting it 😉

**Comment by dimaip** [comment]
It definitely is a very unfriendly behaviour, we should fix it

**Comment by danielkestler** [comment]
Yeah absolutely. But shouldn‘t Neos itself not render this abstract nodetype from the beginning? Normally, there are these „missing nodetype definition“ warnings if I am correct...

**Comment by dimaip** [comment]
They are rendered in order so it would be possible to fix them manually
